mw-webapp/package.json

```diff
@@ -72,11 +72,12 @@
     "@typescript-eslint/parser": "^6.4.0",
     "@vitejs/plugin-react": "^4.0.4",
     "eslint": "^8.47.0",
+    "eslint-plugin-import": "^2.28.1",
```

Comment:
Can't we use https://eslint.org/docs/latest/rules/no-restricted-exports instead of add new dependency?
If you think that we can't use it then explain please why?